htdocs/compta/bank/bankentries_list.php

```diff
@@ -497,8 +499,9 @@
 $sql .= " ".MAIN_DB_PREFIX."bank_account as ba,";
 $sql .= " ".MAIN_DB_PREFIX."bank as b";
 if (is_array($extrafields->attributes[$object->table_element]['label']) && count($extrafields->attributes[$object->table_element]['label'])) $sql .= " LEFT JOIN ".MAIN_DB_PREFIX.$object->table_element."_extrafields as ef on (b.rowid = ef.fk_object)";
-$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."bank_url as bu ON bu.fk_bank = b.rowid AND type = 'company'";
-$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe as s ON bu.url_id = s.rowid";
+$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."bank_url as bu ON bu.fk_bank = b.rowid AND (type = 'company' OR type = 'user')";
+$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe as s ON bu.url_id = s.rowid AND bu.type = 'company'";
```

Comment:
Having 2 left join here is dangerous because if we have bot a link of type thirdparty and type user, the result of sql will be duplicated. I dont think we can solve this in sql so i suggest:
- restore old sql request with only 1  left join on type 'company'.

Later when we need to show the user in the
`if (!empty($arrayfields['bu.label']['checked']))`
If the company is not set, we try to load the user.

Reply:
Indeed, I resolve it. 
Conditions if/else to show user are less clear but I have no other options without touching first sql request.